### ISSUE TITLE
Inline Edit Row Duplicated

### ISSUE THREAD

**Issue by oemad** [hjalmers/angular-generic-table#271]
![screenshot from 2018-07-24 15-48-07](https://user-images.githubusercontent.com/33284809/43142690-4320a736-8f59-11e8-9064-da1c8fe81ebe.png)

For some reason that I can't find, Whenever I enable inline edit all rows are duplicated, so you can see one row with inline edit and another without

**Comment by hjalmers** [comment]
Have you imported the css or sass file in your project?

**Comment by oemad** [comment]
Yes I have, the point is in each cell 2 spans are generated, I tweaked a bit with the css to hide it, but i shouldn't get generated twice in the first place

**Comment by hjalmers** [comment]
No it shouldn't, when the `stack` option is set to true it will add an extra span to the column but it's only for the heading and not the value. Can you reproduce it in [stackblitz](https://stackblitz.com/edit/angular-generic-table-inline-edit) or post an example of your code so I can take a look?
